### DISCUSSION TITLE
Como limpar os teste no MAPOS? E como mudar o numero inicial das OSs[resolvido]?

### DISCUSSION THREAD

**Discussion by SuperPorte** [RamonSilva20/mapos#1559] [Q&A]
Saudações!!
Instalei o MAPOS no meu PC e fiz varios cadastros aleatórioops para teste. agora....
Como limpo os dados de testes para passar a usar o sistema?
EEEE
Como faço para iniciar as OSs em um número maior?
No sistema que uso atualmente, já estou na OS178, queria não voltar na OS01

Muitíssimo obrigado!!

**Comment by Pr3d4dor** [comment]
@bernarcom Basta você limpar o banco de dados.

Para iniciar a OS em um número maior que 1 precisa alterar o auto increment da tabela de OS.

**Reply by SuperPorte** [reply]
Opa!!
Essa limpeza que não sei como faz. :(

Sobre o numero inicial da OS, me informaram assim:
acessar o phpmyadmim;
abrir a base mapos;
clicar na tabela "os";
clicar na aba operacões;
quadro "opões de table";
no campo "AUTO_INCREMENT", digite o numero da próxima OS a ser aberta;
clicar em [EXECUTAR]

![image](https://user-images.githubusercontent.com/68247011/143877866-2b13cf51-21fd-49de-9dda-f818b8041d53.png)


Pra mim caiu como uma luva =D

**Reply by Pr3d4dor** [reply]
@bernarcom basta você excluir e criar o banco de dados mapos ali na esquerda.
